### DISCUSSION TITLE
How do I change the shape of the profile picture border?

### DISCUSSION THREAD

**Discussion by surajp92** [academicpages/academicpages.github.io#1983] [Q&A] [answered]
Is there a way to change the profile pic to square shape instead of oval shape? What do I need to change in _config.ml file? Thank you.

**Comment by kennethbunker** [comment]
Did you find out?

**Comment by senshineL** [comment, marked as answer]
I think this could be done by setting the object author__avatar in _sidebar.scss.
1. comment out the line "border-radius: 50%"
2. set width and height as you like

**Reply by rjzupkoii** [reply]
This is correct, the `border-radius` CSS property in `_sass/_sidebar.scss` controls how rounded the corners are around the avatar. Setting it to 0% will result in a square border while intermediary values [will give you different effects](https://www.w3schools.com/css/css3_borders.asp).
